ports/py-pyqt-builder/portfile.cmake

```diff
@@ -1,8 +1,9 @@
 vcpkg_from_pythonhosted(
     OUT_SOURCE_PATH SOURCE_PATH
     PACKAGE_NAME    PyQt-builder
+    FILE_NAME       pyqt_builder
```

Comment:
just calculated a filename from package_name ?

Reply:
I assume you meant to say "just calculate" without "d".

Compare https://pypi.org/project/PyQt-builder/#files and https://pypi.org/project/PyQt-builder/1.15.4/#files the filenames do not follow a rule.